.vscode/launch.json

```diff
@@ -9,7 +9,17 @@
             "program": "${workspaceRoot}/src/azure-cli/azure/cli/__main__.py",
             "cwd": "${workspaceRoot}",
             "args": [
-                "--help"
+                "acr",
```

Comment:
it's a global vscode setting on the project. can you revert it?

Reply:
Sorry, this will be reverted